integration_tests/jenkins_jobs/01_preparation.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+rm -rf $HOME/.nodef; rm -rf $HOME/.clif
+
+cd $HOME/friday && git fetch && git checkout $TEST_BRANCH && \
+git reset --hard HEAD~5 && git rebase
```

Comment:
What is the purpose of this action?

Reply:
Ensure applying even force-pushed commit

Reply:
Is there any method how to force-apply GIT server commits?

Reply:
```git reset --hard origin/$TEST_BRANCH```